bubblegum/js/src/mpl-bubblegum.ts

```diff
@@ -22,17 +22,16 @@ export function computeDataHash(metadata: MetadataArgs): Buffer {
   return Buffer.from(keccak_256.digest(Buffer.concat([metadataHash, sellerFeeBasisPointsBuffer])));
 }
 
-export function computeCreatorHash(creators: Creator[]): Buffer {
-  let bufferOfCreatorData = Buffer.from([]);
-  let bufferOfCreatorShares = Buffer.from([]);
-  for (const creator of creators) {
-    bufferOfCreatorData = Buffer.concat([
-      bufferOfCreatorData,
-      creator.address.toBuffer(),
-      Buffer.from([creator.share]),
-    ]);
-    bufferOfCreatorShares = Buffer.concat([bufferOfCreatorShares, Buffer.from([creator.verified ? 1 : 0]), Buffer.from([creator.share])]);
```

Comment:
Strange, it looks like this variable was never actually used within the function.